FiveCalls/FiveCalls/FetchUserStatsOperation.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  FetchUserStatsOperation.swift
+//  FiveCalls
+//
+//  Created by Mel Stanley on 1/31/18.
+//  Copyright © 2018 5calls. All rights reserved.
+//
+
+import Foundation
+
+class FetchUserStatsOperation : BaseOperation {
+    
+    var userStats: UserStats?
+    var httpResponse: HTTPURLResponse?
+    var error: Error?
+    
+    override func execute() {
+        
+        // We'll need the user's access token to authenticate the request
+        SessionManager.shared.credentialsManager.credentials {
+            guard $0 == nil else {
+                self.error = $0
```

Comment:
I think we should avoid using implicit parameters like `$0`, `$1` for this method, since it's quite long and give them names for clarity.